rticonnextdds_connector/rticonnextdds_connector.py

```diff
@@ -1419,37 +1371,37 @@ def set_max_objects_per_thread(value):
         _check_retcode(connector_binding.set_max_objects_per_thread(value))
 
     @staticmethod
-    def native_library_build_string():
-        """Returns the build ID of the native libraries as a string
-
-        :return: The build ID of the native libraries used by Connector.
-        :rtype: string
+    def version():
```

Comment:
Make it a property (`version`) or call it `get_version()`.

Reply:
Renamed it to `get_version()`. Combining the `@property` decorator with the `@staticmethod` decorator doesn't seem to work.